styles.css

```diff
@@ -0,0 +1,229 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.button {
+  background: #ffffff;
+  color: #32394b;
+  border: none;
+}
+.splash {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #ffffff;
+  padding: 0.8rem 12rem 0.8rem;
+  border: 2px solid #dde2ff;
+}
+.body-container {
+  padding: 0.5rem 12rem 0.5rem;
+}
+.top-buttons-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1rem 0rem;
+}
+.tasks-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 1rem;
+  grid-auto-flow: dense;
+}
+#input-container {
```

Comment:
we should not add styles to id in css always use class to add styles in an element, this is the convention and also put all the color, font weight values in constants and use them from there